src/vivarium_testing_utils/automated_validation/interface.py

```diff
@@ -315,3 +322,25 @@ def _get_age_groups(self) -> pd.DataFrame:
             # relabel index level age_group_name to age_group
 
         return age_groups.rename_axis(index={"age_group_name": "age_group"})
+
+    def cache_gbd_data(
+        self, data_key: str, data: pd.DataFrame, overwrite: bool = False
+    ) -> None:
+        """Upload the output of a get_draws call to the context given by a data key."""
+        mapped_data: pd.DataFrame = self._format_to_vivarium_inputs_conventions(
+            data, data_key
+        )
+        formatted_data = set_validation_index(mapped_data)
+        self.data_loader.cache_gbd_data(data_key, formatted_data, overwrite=overwrite)
+
+    def _format_to_vivarium_inputs_conventions(
+        self, data: pd.DataFrame, data_key: str
+    ) -> pd.DataFrame:
+        """Format the output of a get_draws call to data schema conventions for the validation context."""
+        data = drop_extra_columns(data, data_key)
+        data = set_gbd_index(data, data_key=data_key)
+        data = vi.scrub_gbd_conventions(data, self.location)
+        data = vi.split_interval(data, interval_column="age", split_column_prefix="age")
```

Comment:
Do we always split age and year intervals every time we call scrub? If so, we can wrap all three.